src/components/common/MapDateSelect/DateSlider.tsx

```diff
@@ -0,0 +1,44 @@
+import { Slider } from '@terraware/web-components';
+import { useMemo } from 'react';
+
+type DateSliderProps = {
+  dates: Date[];
+  onChange: (newDate: Date) => void;
+  initialSelectionIndex: number;
+};
+
+export default function DateSlider(props: DateSliderProps): JSX.Element {
+  const { dates, onChange, initialSelectionIndex } = props;
+
+  const minTime = useMemo(() => {
+    return Math.min(...dates.map((d) => d.getTime()));
+  }, [dates]);
+
+  const maxTime = useMemo(() => {
+    return Math.max(...dates.map((d) => d.getTime()));
+  }, [dates]);
+
+  const marks = useMemo(() => {
+    return dates.map((date) => ({ value: date.getTime() }));
```

Comment:
Just realized that the user has no idea what the intermediate observation-dates are.. this might be a ux issue.. something to discuss with @chudo76 perhaps. User has to complete the selection in order to see the date, instead of making an informed selection.